Files/Views/LayoutModes/ColumnViewBrowser.xaml.cs

```diff
@@ -100,6 +100,13 @@ private void ItemManipulationModel_RemoveSelectedItemInvoked(object sender, List
 
         private void ItemManipulationModel_InvertSelectionInvoked(object sender, EventArgs e)
         {
+            if (IsLastColumnBase)
```

Comment:
Haven't tested the PR but from the changes below it looks like this should be `if (!IsLastColumnBase)`

Reply:
Thanks, fixed